doc/method/method-intro.md

```diff
@@ -87,35 +76,40 @@ The combination S + S' is formed in two steps:
  1. A mapping or _alignment_ step that identifies all
     nodes in S' that can be equated with nodes in S. There will often be nodes
     in S' that cannot be aligned to S.
- 2. A _merge_ step that creates the union, U = S + S', by adding to S the unaligned
-    taxa from S'. The attachment position of unaligned nodes from step 1 is
+ 2. A _merge_ step that creates the combination U = S + S', by adding to S the unaligned
```

Comment:
should this not use the 'workspace' terminology introduced earlier? i.e. "creates the workspace taxonomy W = S + S'"?